### DISCUSSION TITLE
Confluent Cloud and Kafka UI with Helm Charts

### DISCUSSION THREAD

**Discussion by ghost** [provectus/kafka-ui#2953] [Q&A] [answered]
Dear Experts,

Does anyone managed to get Confluent Cloud Configuration with Helm Charts get it working.

I got this working with Docker Compose but facing issues with Helm. I am not sure if I am successfully able to pass KAFKA_CLUSTERS_0_DISABLELOGDIRSCOLLECTION: 'true' via Helm.

I am using values.yml with the command helm install helm-release-name charts/kafka-ui -f .\charts\kafka-ui\values.yaml .

**Comment by ghost** [comment]
I am referring this https://github.com/provectus/kafka-ui/blob/master/helm_chart.md

**Reply by ghost** [reply]
and these https://github.com/provectus/kafka-ui/pull/823, https://github.com/provectus/kafka-ui/pull/823/commits/0ef45bb136434a093c49ac43bd8092aa964ff5d5

**Comment by Haarolean** [comment]
@srikanthprathipati can you provide your `values.yaml` in this case?

**Reply by ghost** [reply]
[values.txt](https://github.com/provectus/kafka-ui/files/10131796/values.txt)
Hi @Haarolean  - please find the values yaml I was trying to use. Apologies for delay in relpying.

**Comment by ghost** [comment]
Seems like we cannot pass KAFKA_CLUSTERS_0_DISABLELOGDIRSCOLLECTION as config map with K8S. I keep getting Error: INSTALLATION FAILED: ConfigMap in version "v1" cannot be handled as a ConfigMap: json: cannot unmarshal bool into Go struct field ConfigMap.data of type string.

Not sure if someone managed to crack helm with Confluent Cloud.

**Comment by ghost** [comment]
Hi @Haarolean , I have managed to crack this with the below hack.
Created a new Image with the Dockerfile as below
`FROM provectuslabs/kafka-ui:master`
`ENV KAFKA_CLUSTERS_0_DISABLELOGDIRSCOLLECTION=true`

Updated values.yaml to use the image and then the below command.

`helm install helm-release-name charts/kafka-ui --set envs.config.KAFKA_CLUSTERS_0_NAME=Integration-DevTest --set envs.config.KAFKA_CLUSTERS_0_BOOTSTRAPSERVERS=gaga:9092 --set envs.config.KAFKA_CLUSTERS_0_PROPERTIES_SECURITY_PROTOCOL=SASL_SSL --set envs.config.KAFKA_CLUSTERS_0_PROPERTIES_SASL_MECHANISM=PLAIN --set envs.config.KAFKA_CLUSTERS_0_PROPERTIES_SASL_JAAS_CONFIG="org.apache.kafka.common.security.plain.PlainLoginModule required username='gaga' password='gaga';" --set envs.config.KAFKA_CLUSTERS_0_PROPERTIES_CLIENT_DNS_LOOKUP=use_all_dns_ips --set envs.config.KAFKA_CLUSTERS_0_SCHEMAREGISTRY=https://api-dv.gaga.com/integrationschemaregistry/dev --set envs.config.KAFKA_CLUSTERS_0_KSQLDBSERVER=https://gaga:443`

it works like a charm now.

**Comment by Haarolean** [comment, marked as answer]
With your config it should look like this:
<img width="604" alt="image" src="https://user-images.githubusercontent.com/1494347/205215158-b6d63dfd-30b9-47ca-b4bb-ebee2090ffd2.png">

**Reply by ghost** [reply]
This works like a charm, thanks for updating here.
